sdks/smart-wallet-sdk/src/types.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Represents a call to a smart contract
+ */
+export interface Call {
+  /** The address of the contract to call */
+  to: string
+  /** The encoded calldata for the call */
+  data: string
+  /** The amount of ETH to send with the call */
+  value: string
+  /** The chain ID for the call (for client-side use) */
+  chainId?: string
+}
+
+/**
+ * Represents a call with advanced options like partial failure handling
+ */
+export interface AdvancedCall extends Call {
+  /** Whether to revert the entire transaction if this call fails */
+  revertOnFailure?: boolean
+}
+
+/**
+ * Parameters for method execution
+ */
+export interface MethodParameters {
```

Comment:
I'm sure we have this defined in another sdk, maybe core?

Reply:
It seems to be defined locally in v3, v4, universal-router-sdk :/ so maybe we can explore moving to sdk-core

Reply:
meh ok ya just leave it :P aha